bin/build

```diff
@@ -45,18 +41,30 @@ if [ -z "$REPO" ]; then
   exit 1
 fi
 
+if [ -n "$LOCAL_RPM" ] && [ -n "$REBUILD_RPM" ]; then
+  echo "Local rpm (-l) and rebuild rpm (-b) options can't be used together"
```

Comment:
Why not?  I think of rebuild_rpm as a subset of local_rpm.  That is rebuild_rpm builds the rpms and writes them into the local-rpm directory, and local-rpm says to use what's in the local-rpm directory.  Or am I minunderstanding?

Reply:
Since rebuild_rpm will delete RPMs on the machine if exists, I thought it will be confusing to allow both options to be used together.

rpm_build implies local_rpm, but the difference is "local RPMs already exist on the machine" vs "local RPMs that will be built".

Reply:
Oh I see...you are concerned about killing the local RPMs.  I can understand that.